include/meta/embeddings/word_embeddings.h

```diff
@@ -108,6 +123,12 @@ class word_embeddings
 
     /// The embeddings matrix
     util::aligned_vector<double> embeddings_;
+
+     /// The target vectors matrix
+    util::aligned_vector<double> target_vectors_;
+
+    /// The context vectors matrix
+    util::aligned_vector<double> context_vectors_;
```

Comment:
This triples the memory requirements for loading embeddings in the common case (target + context mode). I'm not sure I understand why this was added.

To access just target vectors, use the first constructor for `word_embeddings` with the target vector file as the second input stream. To access just the context vectors, use the first constructor of `word_embeddings` with the context vector file as the second input stream.

Is there something I'm missing?